problem_builder/mixins.py

```diff
@@ -261,3 +264,30 @@ def delete_key(self, dictionary, key):
         except KeyError:
             pass
         return dictionary
+
+
+class MakeURLAbsoluteMixin(object):
+    url_pattren = re.compile('<img\s*src="(.*)"')
+
+    def make_url_absolute(self, text):
+        if not text:
+            return text
+
+        try:
+            from static_replace import replace_static_urls
+        except ImportError:
+            return text
+
+        lms_relative_url = replace_static_urls(text, course_id=self.course_id)
```

Comment:
I don't think this code is correct; I think it should be https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/987b7dc424d744633bad55d7d45b7ebdf3105a1d/drag_and_drop_v2/drag_and_drop_v2.py#L828-L845